Common/include/option_structure.hpp

```diff
@@ -1323,6 +1323,20 @@ static const MapType<string, TURBOMACHINERY_TYPE> TurboMachinery_Map = {
   MakePair("AXIAL_CENTRIFUGAL",  AXIAL_CENTRIFUGAL)
 };
 
+/*!
+ * \brief Types of Turbomachinery performance Type.
+ */
+enum TURBO_PERF_KIND{
+  TURBINE = 0,            /*!< \brief Turbine Performance. */
+  COMPRESSOR = 1,         /*!< \brief Compressor Performance. */
+  PROPELLOR = 2           /*!< \brief Propellor Performance. */
+};
+static const MapType<string, TURBO_PERF_KIND> TurboPerfKind_Map = {
+  MakePair("TURBINE",  TURBINE)
+  MakePair("COMPRESSOR",  COMPRESSOR)
+  MakePair("PROPELLOR",  PROPELLOR)
+};
```

Comment:
```suggestion
enum class TURBO_PERF_KIND{
  TURBINE,            /*!< \brief Turbine Performance. */
  COMPRESSOR,         /*!< \brief Compressor Performance. */
  PROPELLOR           /*!< \brief Propellor Performance. */
};
static const MapType<std::string, TURBO_PERF_KIND> TurboPerfKind_Map = {
  MakePair("TURBINE",  TURBO_PERF_KIND::TURBINE)
  MakePair("COMPRESSOR",  TURBO_PERF_KIND::COMPRESSOR)
  MakePair("PROPELLOR",  TURBO_PERF_KIND::PROPELLOR)
};
```
This file no longer has the infamous `using namespace std;` and so you need `std::string` etc.